docs/ffmpeg-params-guide.md

```diff
@@ -0,0 +1,187 @@
+
+### Demuxer plugin available params
+
+| Parameter      | Description                                               | Type             | Default               | Possible Values                                  |
+|----------------|-----------------------------------------------------------|------------------|-----------------------|--------------------------------------------------|
+| `ip_addr`      | Set the remote IP address from which the media data will be received. | String           | `"192.168.96.1"`      |                                                  |
+| `port`         | Set the local port on which the media data will be received. | String           | `"9001"`              |                                                  |
+| `payload_type` | Set the payload type for the media data.                 | String           | `"st20"`              | `"st20"`, `"st22"`, ~~`"st30"`~~, ~~`"st40"`~~, ~~`"rtsp"`~~ |
+| `protocol_type`| Set the protocol type for the media data reception.      | String           | `"auto"`              | `"memif"`, `"udp"`, `"tcp"`, `"http"`, `"grpc"`, `"auto"` |
+| `video_size`   | Set the video frame size.                                | Image size (String) | `"1920x1080"`       |                                                  |
+| `pixel_format` | Set the video pixel format.                              | Pixel format (Enum) | `AV_PIX_FMT_YUV422P10LE` | ~~`AV_PIX_FMT_NV12`~~, `AV_PIX_FMT_YUV422P`, `AV_PIX_FMT_YUV444P10LE`, `AV_PIX_FMT_RGB24`, `AV_PIX_FMT_YUV422P10LE` |
+| `frame_rate`   | Set the video frame rate.                                | Video rate (String or Rational) | `"25"` |                                                  |
+| `socket_name`  | Set the memif socket name for the media data reception.  | String           | `NULL`                |                                                  |
+| `interface_id` | Set the interface ID for the media data reception.       | Integer          | `0`                   |                                                  |
+
+## FFmpeg Media Communications Mesh Muxer Plugin Documentation
+
+The Mesh's Muxer plugin for FFmpeg is designed to handle the transmission of media data over a network using various protocols and payload types. Below are the input parameters that can be configured for the Mesh's Muxer plugin.
+
+### Input Parameters
+
+#### `ip_addr`
+- **Description**: Set the remote IP address to which the media data will be sent.
+- **Type**: String
+- **Default**: `"192.168.96.2"`
+- **Flags**: Encoding parameter
+
+#### `port`
+- **Description**: Set the remote port to which the media data will be sent.
+- **Type**: String
+- **Default**: `"9001"`
+- **Flags**: Encoding parameter
+
+#### `payload_type`
+- **Description**: Set the payload type for the media data.
+- **Type**: String
+- **Default**: `"st20"`
+- **Flags**: Encoding parameter
+- **Possible Values**: `"st20"`, `"st22"`, ~~`"st30"`~~, ~~`"st40"`~~, ~~`"rtsp"`~~
```

Comment:
@Mionsz Only st20 and st22 are supported at this moment.

Reply:
There is a strikethrough 
![image](https://github.com/user-attachments/assets/8a4210bc-d726-4ce2-bee8-d3b7ffc470c3)